syntax = "proto3";

package common.proto;

message RpcMeta {
  optional string service_name = 1;
  optional string method_name = 2;
}
